.github/workflows/cd-spec.yml

```diff
@@ -0,0 +1,24 @@
+name: Spec
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    if: "! contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - name: Begin CI...
+        uses: actions/checkout@v2
+      - name: Use Node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install
+        run: npm i
+
+      - name: Build Spec
+        run: npm run build
+
+      - name: Deploy Spec
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
```

Comment:
We will want to configure this to serve from the `gh-pages` branch... which will be build from `docs` by CI... this keeps your main branch from looking like a web server root.